response.go

```diff
@@ -120,19 +120,22 @@ func (r *Response) RawBody() io.ReadCloser {
 }
 
 func (r *Response) fmtBodyString() string {
-	bodyStr := "***** NO CONTENT *****"
 	if r.body != nil {
+		if len(r.body) > 1000000 {
```

Comment:
What is this `1000000` value?  are you picking random max value to limit it? and why?

Reply:
Yea.. It is arbitrary. What do you suggest? Should I add a `var` so that the user can override if required?

Reply:
Making this value configurable at [client level](https://github.com/go-resty/resty/blob/master/client.go#L68) would be best. Since some user wants to get complete body value.

* Add an un-exported field `debugBodySizeLimit int64` to `Client` and method in the client `SetDebugBodyLimit(sl int64)`
* Initializing `debugBodySizeLimit` default value as `math.MaxInt32` (this is good enough to begin with) at [client instance creation](https://github.com/go-resty/resty/blob/master/default.go#L27) 
* Add corresponding method for default resty client in [default.go](https://github.com/go-resty/resty/blob/master/default.go)

This way it will satisfy possibilities.